common/models/user.js

```diff
@@ -119,11 +120,12 @@ module.exports = function(User) {
    * @returns {Object} The normalized credential object
    */
   User.normalizeCredentials = function(credentials, realmRequired, realmDelimiter) {
+    var caseSensitiveEmail = this.settings.caseSensitiveEmail;
```

Comment:
Move this down under line 127 so the variable doesn't always initialize if `credentials.email` doesn't exist.